src/02-common-principles.md

```diff
@@ -422,13 +422,13 @@ dictionary matches a column name in the TSV file, then that field MUST contain a
 description of the corresponding column, using an object containing the following
 fields:
 
-| Field name  | Definition                                                                                                      |
-| :---------- | :-------------------------------------------------------------------------------------------------------------- |
-| LongName    | Long (unabbreviated) name of the column.                                                                        |
-| Description | Description of the column.                                                                                      |
-| Levels      | For categorical variables: a dictionary of possible values (keys) and their descriptions (values).              |
-| Units       | Measurement units. SI units in CMIXF formatting are RECOMMENDED (see [Units](./02-common-principles.md#units)). |
-| TermURL     | URL pointing to a formal definition of this type of data in an ontology available on the web.                   |
+| Field name  | Definition                                                                                                                   |
+| :---------- | :--------------------------------------------------------------------------------------------------------------------------- |
+| LongName    | OPTIONAL. Long (unabbreviated) name of the column.                                                                           |
+| Description | REQUIRED. Description of the column.                                                                                         |
+| Levels      | RECOMMENDED. For categorical variables: a dictionary of possible values (keys) and their descriptions (values).              |
+| Units       | RECOMMENDED. Measurement units. SI units in CMIXF formatting are RECOMMENDED (see [Units](./02-common-principles.md#units)). |
+| TermURL     | RECOMMENDED. URL pointing to a formal definition of this type of data in an ontology available on the web.                   |
```

Comment:
I think all of these fall in the OPTIONAL/RECOMMENDED range. I'm guessing you marked `Description` as REQUIRED because of 

> that field MUST contain a description of the corresponding column, using an object containing the following
fields

But I interpret that as the object is the description, not that the Description field MUST be present.

Reply:
yes, that was my reasoning. I'd be fine with changing this to recommended.
```suggestion
| LongName    | OPTIONAL. Long (unabbreviated) name of the column.                                                                           |
| Description | RECOMMENDED. Description of the column.                                                                                      |
| Levels      | RECOMMENDED. For categorical variables: a dictionary of possible values (keys) and their descriptions (values).              |
| Units       | RECOMMENDED. Measurement units. SI units in CMIXF formatting are RECOMMENDED (see [Units](./02-common-principles.md#units)). |
| TermURL     | RECOMMENDED. URL pointing to a formal definition of this type of data in an ontology available on the web.                   |
```